### ISSUE TITLE
Add documentation about "Show as expanded" menu item option

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa#3910]
# Motivation

Since we made it so that the "Show as expanded" menu item option actually worked as expected, I have gotten multiple questions about why sub-menu items were not showing up as expected.

# Proposed solution

Write and publish documentation about this option to assist our site owners and editors with understanding how to use this feature.

**Comment by e-marie-w** [comment]
[Draft here, only added 'Show as Expanded' bullet point.](https://sitenow.uiowa.edu/node/211/latest) Didn't want to start something huge or difficult on such a short week so I went rogue for a little while. Also there was a Slack conversation about this the other week so it seemed useful to do. I mostly used suggestions from that conversation. 

I was going to add something about how it only works for up to two sub levels but given the work being done on the mega menu I decided not to. 

Let me know if it should be explained more in-depth or moved to a different location.
